src/client/store/pages/originalDataPoint/actions/updateOriginalDataPoint.ts

```diff
@@ -27,6 +27,7 @@ const putOriginalDataPoint = Functions.debounce(
           countryIso,
           assessmentName,
           cycleName,
+          sectionName: 'extentOfForest',
```

Comment:
don't we need to pass this to getODP as well?

Reply:
We use middleware `canView` for read operations which does not take into account the sectionName

https://github.com/openforis/fra-platform/blob/703a7ffeba04efb8a7570680fb4eebfb590ac6a6/src/server/middleware/auth.ts#L46-L45